membership-attribute-service/conf/routes

```diff
@@ -7,13 +7,17 @@ GET        /user-attributes/me/features                     controllers.Attribut
 GET        /user-attributes/me/mma-digitalpack              controllers.AccountController.digitalPackDetails
 GET        /user-attributes/me/mma-membership               controllers.AccountController.membershipDetails
 GET        /user-attributes/me/mma-monthlycontribution      controllers.AccountController.monthlyContributionDetails
+GET        /user-attributes/me/mma-paper                    controllers.AccountController.paperDetails
 
 POST       /user-attributes/me/membership-update-card       controllers.AccountController.membershipUpdateCard
 OPTIONS    /user-attributes/me/membership-update-card       controllers.AccountController.membershipUpdateCard
 
 POST       /user-attributes/me/digitalpack-update-card      controllers.AccountController.digitalPackUpdateCard
 OPTIONS    /user-attributes/me/digitalpack-update-card      controllers.AccountController.digitalPackUpdateCard
 
+POST       /user-attributes/me/paper-update-card      controllers.AccountController.paperUpdateCard
```

Comment:
I appreciate that this is following the existing pattern, but it would be good if we could look to consolidate all of these endpoints under /me/update-card in the future - as all this is happening at the account level, I don't think we should need a new function for every card update